### PR TITLE
feat: add scriptPayout and scriptOperatorPayout to `protx diff` / CSimplifiedMNListEntry

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
As requested by Ivan

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by QuantumExplorer** [comment]
I would really like it if v18 was feature frozen at this point @PastaPastaPasta

**Comment by UdjinM6** [comment]
We could make `scriptPayout` and `scriptOperatorPayout` mem-only if we only need these fields in rpc and don't want to change `mnlistdiff` format 3e7d25ed. But it would still add mns with updated `scriptPayout` and/or `scriptOperatorPayout` to `mnList` (because of changes in `operator==`), so messages will be larger in these cases. Also, (light) nodes won't see any changes comparing to the data they already have and they might get confused. This can be fixed by tweaking `BuildSimplifiedDiff` 310114c7dd and extending `protx diff` params 055bad15d4 (to make sure we can get both results - 0. (default) matching p2p message and 1. with additional fields and items).

File: src/evo/simplifiedmns.h [hunk 1]
```diff
@@ -56,7 +60,9 @@ class CSimplifiedMNListEntry
                 obj.service,
                 obj.pubKeyOperator,
                 obj.keyIDVoting,
-                obj.isValid
+                obj.isValid,
+                obj.scriptPayout,
+                obj.scriptOperatorPayout
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/evo/simplifiedmns.h:65]: What if we just remove it?

[the remark this answers is not in the document]
Comment (UdjinM6) [reply src/evo/simplifiedmns.h:65]: remove what? serialization? i.e. like in https://github.com/dashpay/dash/commit/3e7d25ed0c0e192b484222c221dd1ca130dcf311? see https://github.com/dashpay/dash/pull/4740#issuecomment-1079943224 then :)

[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/evo/simplifiedmns.h]: Hmm, good idea :P see latest

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Comment by UdjinM6** [comment]
@PastaPastaPasta do we need this in v18? pls set the milestone (and "backport" label if needed)

**Comment by PastaPastaPasta** [comment]
No we do not need this in v18 and I'd prefer adding another rpc change in this version.

**Pull request merged**